crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -456,5 +456,257 @@ static_assert(not is_subtype_of(Intersection[Unknown, int], int))
 static_assert(not is_subtype_of(tuple[int, int], tuple[int, Unknown]))
 ```
 
+## Callable
+
+The general principle is that a callable type is a subtype of another if it's more flexible in what
+it accepts and more specific in what it returns.
+
+References:
+
+- <https://typing.python.org/en/latest/spec/callables.html#assignability-rules-for-callables>
+- <https://typing.python.org/en/latest/spec/callables.html#assignment>
+
+### Return type
+
+Return types are covariant.
+
+```py
+from typing import Callable
+from knot_extensions import is_subtype_of, static_assert
+
+static_assert(is_subtype_of(Callable[[], int], Callable[[], float]))
+static_assert(not is_subtype_of(Callable[[], float], Callable[[], int]))
+```
+
+### Parameter types
```

Comment:
Might be good to add some tests with more than 0-1 parameters?

Reply:
Added tests with multiple parameters.